cypress/e2e/MyOwnTests/forgot-password.spec.js

```diff
@@ -0,0 +1,28 @@
+/// <reference types = "cypress" />
+import {ForgotPassword} from '../../../pages/Forgot-password.page'
+import {RegisterPage} from '../../../pages/Register.page'
+import {LoginPage} from '../../../pages/Login.page'
+describe('Forgot password page tests with PageObject', ()=> {
+const forgotPassword = new ForgotPassword()
+const registerPage = new RegisterPage ()
+const loginPage = new LoginPage()
+
+it('Email validation', ()=>{
```

Comment:
```suggestion
it('Improper email validation (negative test)', ()=>{
```
З назви тесту не зрозуміло, що тут буде тестуватися